### ISSUE TITLE
Enhance model.validate with viewName

### ISSUE THREAD

**Issue by dzhitomirsky-sc** [OpusCapita/react-crudeditor#231]
**Intro**
Now `model.validate` takes the only parameter 'instance':
```
model: {
...
 validate: instance => {

 }
}
```
In certain cases it is needed to know from which validation is called (e.g. validating PK unique key).

**ToDo**
So, please extend validation function with the next api:

```
model: {
 validate: ({instance, viewName}) => {

 }
}

```

**Comment by amourzenkov-sc** [comment]
Relevant Mocha tests still need correction. They are `describe.skip`-ed now.

**Comment by estambakio-sc** [comment]
This was merged into #230 with fixed tests.

**Comment by amourzenkov-sc** [comment]
Change the function arguments to

```
model: {
  validate: ({persistentInstance, formInstance, viewName}) => {
    ...
  }
}
```
